evacuees/src/UI/embc-evacuee/src/app/sharedModules/components/error-screen/error-screen.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="main row">
+    <div class="col-md-3 image-content">
+        <img src="../../../../assets/images/warning-icon2.png">
+    </div>
+    <div class="col-md-9 text-content">
+        <h1 style="font-size: 200%;"><b>Something went wrong</b></h1>
+        <p style="margin: 0; font-size: 120%;">Sorry for the inconvenience. At this moment we cannot process your
```

Comment:
Change the meesage to  => The service is temporarily unavailable. Please try again later